docs/src/manual/misc/FacElem.md

```diff
@@ -6,30 +6,30 @@ CurrentModule = Hecke
 
 In many applications in number theory related to the multiplicative
 structure of number fields, interesting elements, e.g. units,
-are extremely large when written wrt. to a fxied basis for the field:
+are extremely large when written with respect to a fixed basis for the field:
 for the fundamental unit in $Q[\sqrt d]$ it is known that the coefficients
-wrt. the canonical basis $1, \sqrt d$ can have $O(\exp \sqrt d)$ many digits.
+with respect to the canonical basis $1, \sqrt d$ can have $O(\exp \sqrt d)$ many digits.
 All currently known, fast methods construct those elements as power
 products of smaller elements, allowing the computer to handle them.
 
 Mathematically, one can think of factored elements to formally
-live in the ring $Z[K]$ the group ring of the non-zero field
-elements. Thus elements are of the form $ \prod a_i^{e_i}$ where
-$a_i$ are elements in $K$, typically _small_ and the $e_i\in Z$ are frequently
+live in the ring $\mathbb{Z}[K^*]$ the group ring of the non-zero field
+elements. Thus elements are of the form $\prod a_i^{e_i}$ where
+$a_i$ are elements in $K$, typically _small_ and the $e_i\in \mathbb{Z}$ are frequently
 large exponents. We refer to the $a_i$ as the *base* and the $e_i$ as the
 *exponents* of the factored element.
 
 Since $K$ is, in general, no PID, this presentation
```

Comment:
I don't understand this line: $K$ is a field, so of course it is a PID. Perhaps $\mathbb{Z}[K^*]$ is meant? But also no assumption is made about the $a_i$ so of course there is no uniqueness. Or maybe the some order in $K$ is meant? Still...?!?

Reply:
I think this part of the sentence is mixing up a few things and should probably be removed.